app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,19 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          _.each(scope.pod.spec.volumes, function(volumeData) {
+            if (!volumeData.persistentVolumeClaim) {
+              return;
+            }
+            volumeDataSets.push({
+              id: "filesystem/usage",
+              label: volumeData.name,
+              data: []
+            });
```

Comment:
It still looks to me like the metrics request for each volume will return the same data, so you'll have a multiline chart with identical lines if you have more than one volume. Can you check this case?

If you can show usage per volume, great. If we can't, we should make this a simple area chart with total usage.